src/GafferScene/Preview/RendererAlgo.cpp

```diff
@@ -512,6 +513,21 @@ struct LocationOutput
 				updatedAttributes->members()[g_setsAttributeName] = boost::const_pointer_cast<InternedStringVectorData>( setsAttribute );
 			}
 
+			// TODO: clean up
+			CompoundObject::ObjectMap::const_iterator it = updatedAttributes->members().find( "linkedLightsSet" ); // TODO: naming needs to be determined
```

Comment:
I have fixed it in one of the latest commits. I guess my reasoning was that we are pretty much done with those `attributes` after we copied over everything in https://github.com/GafferHQ/gaffer/blob/master/src/GafferScene/Preview/RendererAlgo.cpp#L507. We could just operate on the new data after that, I guess, but the latest code will use `attributes` now :)